recipes/debugpy/LICENSE.txt

```diff
@@ -0,0 +1,24 @@
+    debugpy
```

Comment:
No need to include the LICENSE, it should be part of the `sdist`: https://github.com/microsoft/debugpy/blob/4715925a00337b567b45aa1103addb75a5c8a5cd/MANIFEST.in#L1 The `ThirdPartyNotices.txt` isn't though, care to make a PR?